keras_cv/layers/object_detection/multi_class_non_max_suppression.py

```diff
@@ -57,13 +57,13 @@ def __init__(
         self.max_detections_per_class = max_detections_per_class
         self.built = True
 
-    def call(self, box_prediction, confidence_prediction):
+    def call(self, box_prediction, class_prediction):
```

Comment:
Maybe I'm overly indoctrinated in the small CL concept, but IMO it would be better to break the NMS + LabelEncoder changes to separate PRs. Up to you.

Reply:
Lets leave this one for now, but I am happy to be more mindful of this going forwards.